src/home/views/index.tsx

```diff
@@ -25,6 +37,9 @@ const HomeSection = () => {
           topProducts={maybe(() =>
             data.productTopToday.edges.map(edge => edge.node)
           )}
+          channelChoices={channelChoices}
+          channelValue={channelChoice}
+          onChannelChange={useCallback(value => setChannelChoice(value), [])}
```

Comment:
I believe this breaks rule of hooks
https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level